buildSrc/src/main/kotlin/com/datadog/gradle/config/MavenConfig.kt

```diff
@@ -6,6 +6,109 @@
 
 package com.datadog.gradle.config
 
+import org.apache.maven.model.Developer
+import org.apache.maven.model.License
+import org.apache.maven.model.Model
+import org.apache.maven.model.Scm
+import org.gradle.api.Project
+import org.gradle.api.plugins.MavenRepositoryHandlerConvention
+import org.gradle.api.tasks.Upload
+import org.gradle.internal.impldep.org.sonatype.aether.repository.Authentication
+import org.gradle.internal.impldep.org.sonatype.aether.repository.RemoteRepository
+import org.gradle.jvm.tasks.Jar
+import org.gradle.plugins.signing.SigningExtension
+
 object MavenConfig {
     const val GROUP_ID = "com.datadoghq"
+    const val PASSWORD = "ossrhPassword"
+    const val USER_NAME = "ossrhUsername"
```

Comment:
better to remove these 2 even if they are not real and not used, because any security scanner will flag them anyway.